src/test/python_tests/test_code_actions.py

```diff
@@ -257,6 +257,16 @@ class Banana(object):
     print(f"{city} has population {population}.")
 """,
             """for city, population in data.items():
+""",
+        ),
+        (
+            "W1510:subprocess-run-check",
+            """
+import subprocess
+
+proc = subprocess.run(["ls"])
+""",
+            """proc = subprocess.run(["ls"], check=False)
```

Comment:
What happens for multiline?
```
proc = subprocess.run(
    ["ls"],
    cwd=".",
    stdout=subprocess.PIPE,
    stderr=subprocess.PIPE,
)
```

Reply:
Good catch.
Might take me a little bit to figure this one out Not sure I have done a multiline fix before.

Reply:
@karthiknadig , man am i stuck on this.... looks like this won't fit into the regex pattern easily. 

I feel like there should be a simpler way for us to add a new parameter to a function call like subprocess.run, any examples of doing something like this anywhere else that we can go off of?

Reply:
@brettcannon Any recommendation on how we can do this? 

Other than actually using a parser to find the right spot, I don't see a way to do this correctly. I honestly think that these change recommendations could generated from pylint itself. Thank all we would have to do is surface them via code actions.

/cc @DanielNoord

Reply:
We don't have a way to autofix code directly in pylint right now but we have an open issue for this: https://github.com/pylint-dev/pylint/issues/7438. The fix is certainly going to be easier in the context of pylint where we sometime already display the suggestion in the message... vs by regex here :) We also have a lot on our plate so any help would be greatly appreciated and reviewed promptly.

Reply:
For a keyword argument, you could find the closing `)` and then prepend to it, but that's assuming you aren't dealing with any nested parentheses. Otherwise you're wading into using the `ast` module to parse the code, update the AST, and then regenerate the string representation (which even using [`ast.unparse()`](https://docs.python.org/3/library/ast.html#ast.unparse) doesn't not guarantee an exact result).

Basically you're playing with 🔥 on this one.